auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/Parcelables.java

```diff
@@ -168,6 +169,18 @@ static void appendReadValue(CodeBlock.Builder block, AutoValueParcelExtension.Pr
     }
   }
 
+  static void appendReadValueWithTypeAdapter(CodeBlock.Builder block, AutoValueParcelExtension.Property property, final FieldSpec adapter) {
```

Comment:
Can you rename this method to keep it consistent with the others?  We've got `writeValue`, `writeValueWithTypeAdapter` and `appendReadValueWithTypeAdapter`.  It can just be `readValueWithTypeAdapter`.